app/jobs/suitable_for_early_allocation_email_job.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class SuitableForEarlyAllocationEmailJob < ApplicationJob
+  queue_as :mailer
+
+  EQUIP_URL = 'https://equip-portal.rocstac.com/CtrlWebIsapi.dll/?__id=webDiagram.show&map=0%3A9A63E167DE4B400EA07F81A9271E1944&dgm=4F984B45CBC447B1A304B2FFECABB777'
+
+  def perform
+    offenders = EarlyAllocation.where(created_within_referral_window: false).where.not(outcome: 'ineligible').pluck(:nomis_offender_id).uniq
+
```

Comment:
It would be nice to make this a scope, so that EarlyAllocation encapsulates this behaviour rather than the client knowing about it - it's nice if 'outcome' is considered private to EarlyAllocation even if it's not enforceable by rails